src/components/Sections/SectionProgress.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Icon, Label, Progress } from 'semantic-ui-react'
+import { ApplicationProgress, ReviewProgress, SectionState } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const SectionProgress: React.FC<SectionState> = ({ reviewProgress, reviewAction }) => {
```

Comment:
this is review specific right ? Should it have review specific name ?

Reply:
Yep, good idea

Reply:
A little verbose naming, but at least that what the component is about `ReviewStatusOrProgress` Done: 1bd1d3b